studio/schemas/documents/historyYear.js

```diff
@@ -0,0 +1,22 @@
+export default {
+    name: 'historyYear',
+    title: 'History Year',
+    type: 'document',
+    fields: [
+        {
+            name: 'year',
+            title: 'Year',
+            type: 'string',
+        },
+        {
+            name: 'historyText',
+            title: 'History Text',
+            type: 'text',
+        },
+        {
+            name: 'image',
+            title: 'Image',
+            type: 'image',
+        },
```

Comment:
Lets create a custom image object for all images that aren't background images (similar to [this](https://github.com/bckrm/gianna-biscontini/blob/main/studio/schemas/objects/mainImage.js))

Reply:
@asasmith sure thing: I was looking in the documentation last night about this. When do we use a custom image vs. the image type bootstrapped in sanity?

Reply:
or a custom block type vs the block type?